### PR TITLE
Enable Go Modules.

### LINKED ISSUE #28: Go Modules Support
As discussed in #27, and other breaking changes discussions such as #21, we should maintain our own tagging, as well as go module support for our dependencies.

Currently, tags do exist up to `v0.1.2`, however, since then there have been breaking changes in #21 and another proposed in #27.

Because this library is still, essentially v0, I think it may be OK to push `v0.1.3` pointing to latest commit even though it would then tag a breaking change. Once #27 is resolved, and if there's no immediate list of breaking changes, `v1.0.0` could be released.

Along with `v0.1.3` tag, we can add Go modules commit, as I see no reason not to add that.

### PR DESCRIPTION
**Pull request by wlynch**
See https://golang.org/doc/go1.13#modules for more details.

Also pulls in some auto-formatting changes.

Fixes #28

### CODE REVIEW

**Comment by bradleyfalzon** [comment]
Do you have time to resolve the conflicts?

**Comment by wlynch** [comment]
Updated!

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module github.com/bradleyfalzon/ghinstallation
+
+go 1.13
+
+require (
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/google/go-cmp v0.3.1
+	github.com/google/go-github/v28 v28.1.1
```
Comment (bradleyfalzon) [inline go.mod]: Do you know why the import path changed for `go-github` and not `jwt-go`?

Comment (bradleyfalzon) [reply go.mod]: Don't worry, I see now, they have v2+ but don't yet support Go Modules.

**Pull request merged**